Java/core/src/main/java/com/amazon/randomcutforest/sampler/AbstractStreamSampler.java

```diff
@@ -66,6 +66,34 @@
      */
     protected final double initialAcceptFraction;
 
+    /**
+     * a function that computes the probability of admittance of a new value when
+     * the sampler is not full Note that a value can always be admitted if it has a
+     * weight smaller than some sampled value
+     *
+     * this function provides a mechanism for different trees to smoothly diverge --
+     * most previous versions corresponded to initialFraction = 1, and the samplers
+     * only diverge after all of them store all the first sampleSize points. In
+     * contrast the method (which can be changed in a subclass) admits the first
+     * initialFraction * sampleSize number of points and then becomes a monotonic
+     * decreasing function.
+     *
+     * This function is supposed to be a parallel to the outputAfter() setting in
+     * the forest which controls how scores are emitted
+     * 
+     * @param currentSize the current size of the sampler
+     * @return the probability of admitting the next point
+     */
+    protected double compute_fraction(int currentSize) {
```

Comment:
This needs a better name, but I'm having trouble coming up with a good one. `initialRegimeAcceptDecision` is a bit of mouthful. Maybe just `initialAcceptDecision`?

Reply:
initialAcceptDecision sounds good.

Reply:
initialAcceptProbability is perhaps more accurate